### DISCUSSION TITLE
update lxc script

### DISCUSSION THREAD

**Discussion by frezeen** [tteck/Proxmox#625] [Ideas]
about update lxc script a have a 102 lxc with openwrt and whern i start the script ill got this error:
lxc-attach: 102: ../src/lxc/attach.c: lxc_attach_run_command: 1839 No such file or directory - Failed to exec "bash"

can u add a ID skip request at start of script?
somethinks like:
what ID u want to skip? 102 103 104 ...

**Comment by tteck** [comment]
done
